src/commands/bitfield.json

```diff
@@ -44,84 +44,91 @@
+                        "name": "encoding_offset",
+                        "type": "block",
+                        "optional": true,
+                        "arguments": [
+                            {
+                                "name": "encoding",
+                                "type": "string"
+                            },
+                            {
+                                "name": "offset",
+                                "type": "integer"
+                            }
+                        ]
                     },
                     {
-                        "name": "increment",
-                        "type": "integer"
-                    }
-                ]
-            },
-            {
-                "token": "OVERFLOW",
-                "name": "wrap_sat_fail",
-                "type": "oneof",
-                "optional": true,
-                "arguments": [
-                    {
-                        "name": "wrap",
-                        "type": "pure-token",
-                        "token": "WRAP"
+                        "token": "SET",
+                        "name": "encoding_offset_value",
+                        "type": "block",
+                        "optional": true,
+                        "arguments": [
+                            {
+                                "name": "encoding",
+                                "type": "string"
+                            },
+                            {
+                                "name": "offset",
+                                "type": "integer"
+                            },
+                            {
+                                "name": "value",
+                                "type": "integer"
+                            }
+                        ]
                     },
                     {
-                        "name": "sat",
-                        "type": "pure-token",
-                        "token": "SAT"
+                        "token": "INCRBY",
+                        "name": "encoding_offset_increment",
+                        "type": "block",
+                        "optional": true,
+                        "arguments": [
+                            {
+                                "name": "encoding",
+                                "type": "string"
+                            },
+                            {
+                                "name": "offset",
+                                "type": "integer"
+                            },
+                            {
+                                "name": "increment",
+                                "type": "integer"
+                            }
+                        ]
                     },
                     {
-                        "name": "fail",
-                        "type": "pure-token",
-                        "token": "FAIL"
+                        "token": "OVERFLOW",
```

Comment:
should OVERFLOW be outside the `oneof`? and just `optional`?
i guess that's not how the parser of the command itself, works, but more logical right?

Reply:
Should belong only to SET and INCRBY - resolved in 56613896e